cinema/models.py

```diff
@@ -1,10 +1,39 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=63)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField(
+        Actor,
+        related_name="movies"
+    )
+    genres = models.ManyToManyField(
+        Genre,
+        related_name="genres"
```

Comment:
if you have a Genre object, you'd use genre_object.genres.all() to get all the movies of that genre. This can be confusing, as the naming suggests you're getting genres, not movies.

so better to use related_name="movies", the same as for actors